app/views/govuk_component/analytics_meta_tags.raw.html.erb

```diff
@@ -9,6 +9,8 @@
 
   meta_tags["govuk:format"] = content_item_hash[:document_type] if content_item_hash[:document_type]
   meta_tags["govuk:schema-name"] = content_item_hash[:schema_name] if content_item_hash[:schema_name]
+  meta_tags["govuk:publishing-state"] = details_hash[:publishing_state] if details_hash[:publishing_state]
+  meta_tags["govuk:current-language"] = content_item_hash[:locale] if content_item_hash[:locale]
```

Comment:
I'm not sure how that is being populated. If I search for it here: https://github.com/alphagov/static/search?utf8=%E2%9C%93&q=lang%3D&type= 

I don't get any satisfying results, except for the print version. 

It might be that the content_item_hash["locale"] is more accurate than the <html lang="en">, unless there's another place I haven't noticed where the <html lang="en"> is being set.